build/minikube/run-local.sh

```diff
@@ -10,4 +10,6 @@ echo "Using KUBECONFIG '$KUBECONFIG'"
 kubectl create ns ${NAMESPACE} || true
 kubectl apply -f deploy/crds/infinispan.org_infinispans_crd.yaml -n ${NAMESPACE}
 kubectl apply -f deploy/crds/infinispan.org_caches_crd.yaml -n ${NAMESPACE}
+kubectl apply -f deploy/crds/infinispan.org_backups_crd.yaml -n ${NAMESPACE}
```

Comment:
There are a lot of places need to be added new CRD's
- build/run-local.sh
- build/run-okd.sh
- build/push-okd4.sh
- build/crds-gen.sh
- etc.

Reply:
Sure. For now I have only added the bits I've need for local dev. I'll update